### DISCUSSION TITLE
[Feature Request]: Create Pages for Movies, Books & Chapters, Spells and Potions

### DISCUSSION THREAD

**Discussion by danielschuster-muc** [danielschuster-muc/potter-db#171] [Ideas]
### Project

Website (Potter DB Search)

### Describe the feature you want to request

Currently you can only search for characters. There should be also pages for Books, Chapters, Movies, Spells and Potions.

### (Optional) Describe a solution for this feature

Create Pages in the style of the [Character List](https://potterdb.com/characters) and [Detailed Character Page](https://potterdb.com/characters/harry-potter). 
You can also suggest mockups below if you have any ideas for it, but don't know how to implement those pages.

**Comment by danielschuster-muc** [comment]
I've already implemented the movie pages by self, but there are 2 new feature requests, one for Spells (#210) and one for Potions (#209), if anyone wants to work on those.

**Reply by eduardconstantin** [reply]
A page for creatures could be also useful.

**Reply by danielschuster-muc** [reply]
Is already included in the characters page, e.g. [Hungarian Horntail](https://potterdb.com/characters?q=Hungarian+Horntail).

**Reply by eduardconstantin** [reply]
Well creatures are not really characters, don’t you think those categories should be separated?

**Reply by danielschuster-muc** [reply]
Nah, I think we should not seperate those in 2 categories.

**Reply by eduardconstantin** [reply]
Fine
